src/ZipRangeReader.php

```diff
@@ -224,7 +226,7 @@ private function populateFiles()
                 continue;
             }
 
-            $this->files[] = array(
+            $this->files[$file_name] = array(
                 "file_name" => $file_name,
```

Comment:
Should we also keep the original `Index` around once inverted here?

Reply:
I don't think it's needed - it was just the auto-assigned integer index that PHP gave to a keyless array, and we didn't actually use it anywhere. If required, it could be reobtained with `array_values()`.

Reply:
Makes sense!